src/QsCompiler/CSharpGeneration/EntryPoint.fs

```diff
@@ -159,11 +254,11 @@ let private submitMethod context entryPoint =
         ``(``
         [
             param parseResultParamName ``of`` (``type`` "System.CommandLine.Parsing.ParseResult")
-            param settingsParamName ``of`` (``type`` (driverNamespace + ".AzureSettings"))
+            param settingsParamName ``of`` (``type`` (driverNamespace + ".Azure.AzureSettings"))
         ]
         ``)``
         [ ``public`` ]
-        (Some(``=>`` (ident (driverNamespace + ".Azure") <.> (ident "Submit", args))))
+        (Some(``=>`` (ident (driverNamespace + ".Azure.Azure") <.> (ident "Submit", args))))
```

Comment:
I realize this is just mirroring the class namespace/class naming on the runtime repo, but *.Azure.Azure.* seems like an odd choice; I'll take a look at the runtime repo.

Reply:
Having looked at the runtime code, I don't have a better naming idea either.